### ISSUE TITLE
chef_gem rest-client WARN

### ISSUE THREAD

**Issue by ccasey** [adamsb6/s3_file#48]
chef client 12.1.1

```
WARN: chef_gem[rest-client] chef_gem compile_time installation is deprecated
WARN: chef_gem[rest-client] Please set `compile_time false` on the resource to use the new behavior.
WARN: chef_gem[rest-client] or set `compile_time true` on the resource if compile_time behavior is required.
```

from s3_file/recipes/dependencies.rb

**Comment by ccasey** [comment]
https://www.chef.io/blog/2015/02/17/chef-12-1-0-chef_gem-resource-warnings/

**Comment by joekiller** [comment]
Okay thanks for the report.

**Comment by joekiller** [comment]
This should be fixed by #57
